_posts/blog/2020-04-07-mixnet-production-readiness.md

```diff
@@ -0,0 +1,65 @@
+- A way to monitor the health of the network and ensure its resiliency, such as by identifying and kicking out misbehaving nodes. For example, nodes who are performing DDOS attacks against the network and consequently preventing client use.
+- A sufficiently active community and set of contributors to perform software maintenance and general updates and coordination.
+- Application support- the ability for applications to interact with the network, forward and receive traffic sent through the network, and keep locally-held information about the network up to date.
+
+These are just a few examples; this list is not exhaustive.
+
+## What needs to be in place in order for applications to “just start sending their traffic over a mixnet”? Where are we today with usable tools?
+
+Most importantly, it is important to acknowledge that no mixnet currently exists that is of sufficient size to support a large number of users and high amounts of network traffic. However, there are exciting developments such that one could exist in the future (one that I personally am rooting for is [Nym][nym]; their team has well-respected senior researchers in the field of network privacy). However, practically, a lot of growth and maturity is needed before such networks can be used at scale.
+
+Another important consideration to keep in mind is the size of anonymity sets for any anonymity network. Even though mixnets inject latency, the size of the mixnet’s user base still impacts the possible protections for users of that mixnet. So even if such a mixnet did exist, the amount of traffic and number of users directly correspond to the size of the anonymity set protecting each user. In the case of anonymity networks, fewer options is truly better.
+
+## What are the main takeaways I should come away with regarding mixnets and their production-readiness?
+
+First, mixnet designs provide increased security against a global adversary seeking to perform end-to-end timing correlation attacks, whereas low-latency networks like Tor do not consider such an adversary in its threat model.
+
+Second, such a network currently does not exist that can be used at scale, or with as large of an anonymity set as the Tor network.
+
+Third, mixnets do not unilaterally provide better security against *all other kinds of adversaries* than non-mixnet anonymity networks. Other hard security problems still exist, such as securely distributing network information using a sufficient root of trust mechanism, and stable community involvement to perform ongoing tasks like software maintenance.
+
+Finally, network anonymity is only one small part within a larger setting that defines how users are protected in their day-to-day interactions online. So when designing for the safety and privacy of users online, let’s take a step back and truly consider every factor, and strive to use every tool at our disposal to design and build systems that ensure the respect and privacy of our users.
```

Comment:
Love this. There might even be a "every tool, not just the hammers" joke lying around...

Reply:
🛠️